src/index.js

```diff
@@ -1,63 +1,84 @@
-import threepoints from './media/threepoints.png';
-import push from './media/push.png';
-import sync from './media/sync.png';
 import './style.css';
+import threepoints from './assets/threepoints.png';
+import imagetrash from './assets/trash.png';
 
-const list = [
+const tasks = [
   {
-    description: 'Go to the Gym',
-    completed: 'false',
-    index: '1',
+    description: 'Go to Gym',
+    completed: false,
+    index: 0,
   },
   {
-    description: 'Take a cold shower',
-    completed: 'false',
-    index: '2',
+    description: 'Do breackfast',
+    completed: false,
+    index: 1,
   },
   {
-    description: 'Go for lunch',
-    completed: 'false',
-    index: '2',
-  },
-  {
-    description: 'Go to Yoga',
-    completed: 'false',
-    index: '2',
+    description: 'Take a shower',
+    completed: false,
+    index: 1,
   },
```

Comment:
- [ ] It would be nice if you could remove all the hardcoded todos inside this array as stated in the project requirement. Kindly remove them all and have an empty Todo list to which you will be adding. 👍🏽